src/compiler/utilities.ts

```diff
@@ -9788,7 +9788,7 @@ export function isValidBigIntString(s: string, roundTripOnly: boolean): boolean
     if (s === "") return false;
     const scanner = createScanner(ScriptTarget.ESNext, /*skipTrivia*/ false);
     let success = true;
-    scanner.setOnError(() => success = false);
+    scanner.setOnError(() => void (success = false));

```

Comment:
What's going on here?

Reply:
```ts
scanner.setOnError(() => {
    success = false;
});
```
is what I intended; writing this way was probably because both that I did a search of `void (` and found usage in the code, and I wanted to keep the original (inline) code style. I suppose y'all have a strong preference on the multiline style?